### PR TITLE
Mirror Chrome -> Webview for api/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is created by a simple script that mirrors the compatibility data of all APIs from Chrome to Webview when it is set to "null". This should help reduce inconsistencies in the browser data.

### CODE REVIEW

File: api/PublicKeyCredentialCreationOptions.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": null
+            "version_added": "67"
```
Comment (jpmedley) [inline api/PublicKeyCredentialCreationOptions.json]: PublicKeyCredential [isn't supported in WebView](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt?l=88). It's creation options wouldn't be either.

File: api/PublicKeyCredentialRequestOptions.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": null
+            "version_added": "67"
```
Comment (jpmedley) [inline api/PublicKeyCredentialRequestOptions.json]: See above.

**Comment by vinyldarkscratch** [comment]
Thanks for your review, @jpmedley, I’ll make the appropriate changes.  Have a good vacation or otherwise!

**Comment by vinyldarkscratch** [comment]
I went ahead and made the changes based upon this review and the one in the Chrome Android mirroring, so this is all set, I'd say!  Since you've already technically approved of the changes, I'll go ahead and merge this.

**Pull request merged**